lume_model/torch/model.py

```diff
@@ -13,7 +13,12 @@
 class PyTorchModel(BaseModel):
     """The PyTorchModel class is used for the loading and evaluation of online models.
     It is  designed to implement the general behaviors expected for models used with
-    the pytorch lume-model tool kit."""
+    the pytorch lume-model tool kit.
+
+    It is assumed that these models will not be trained and will instead be updated
```

Comment:
I would remove the calibration layers comment, a general user wouldn't know what `calibration layers` refers to